JavaFXUserInterface/src/main/java/Controllers/SystemTypeInfoController.java

```diff
@@ -3,22 +3,20 @@
 import Utilities.SystemTypeList;
 import Utilities.TextConstants;
 import Utilities.UIUtilities;
-import javafx.beans.value.ObservableValue;
+import external.AssetTypeDAOImpl;
 import javafx.fxml.FXML;
 import javafx.fxml.Initializable;
 import javafx.scene.control.*;
 import javafx.scene.image.Image;
 import javafx.scene.image.ImageView;
 import javafx.scene.input.MouseEvent;
-import local.AssetTypeDAOImpl;
 
 import java.net.URL;
-import java.text.DecimalFormat;
 import java.util.Optional;
 import java.util.ResourceBundle;
 
 public class SystemTypeInfoController implements Initializable {
-    private final String SYSTEM_TYPE_LIST = "/Utilities.SystemTypeList";
+    private final String SYSTEM_TYPE_LIST = "/SystemTypeList";
```

Comment:
This is the second time it is declared. Would be better to move it in the constants file and call it from there.